recipe/meta.yaml

```diff
@@ -20,38 +17,48 @@ source:
     - gvconfig_libdir.patch  # [unix]
 
 build:
-  number: 7
+  number: 8
   detect_binary_files_with_prefix: True  # [unix]
 
 requirements:
   build:
-    - cairo 1.14.*             # [linux]
-    - expat 2.2.*                   # [linux]
-    - libpng >=1.6.32,<1.6.35  # [unix]
-    - libtiff >=4.0.8,<4.0.10   # [unix]
-    - libtool                  # [linux]
-    - freetype 2.8.1             # [unix]
-    - jpeg 9*                  # [unix]
-    - pango 1.40.*             # [linux]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
     - pkg-config               # [osx]
-    - zlib 1.2.11               # [unix]
+  host:
+    - cairo                    # [linux]
+    - expat                    # [linux]
+    - libpng                   # [unix]
+    - libtiff                  # [unix]
+    - libtool                  # [linux]
+    - freetype                 # [unix]
+    - jpeg                     # [unix]
+    - pango                    # [linux]
+    - zlib                     # [unix]
+    - libxcb                   # [linux]
+    - xorg-libxrender          # [linux]
+    - xorg-libxpm              # [linux]
   run:
-    - cairo 1.14.*             # [linux]
-    - expat 2.2.*                   # [linux]
-    - libpng >=1.6.32,<1.6.35  # [unix]
-    - libtiff >=4.0.8,<4.0.10   # [unix]
+    - cairo                    # [linux]
+    - expat                    # [linux]
+    - libpng                   # [unix]
+    - libtiff                  # [unix]
     - libtool                  # [linux]
-    - freetype 2.8.1             # [unix]
-    - jpeg 9*                  # [unix]
-    - pango 1.40.*             # [linux]
-    - zlib 1.2.11               # [unix]
+    - freetype                 # [unix]
+    - jpeg                     # [unix]
+    - pango                    # [linux]
+    - zlib                     # [unix]
+    - libxcb                   # [linux]
+    - xorg-libxrender          # [linux]
+    - xorg-libxpm              # [linux]
 
 test:
   commands:
     - dot -V
     - neato -?  # [unix]
     - conda inspect linkages $PKG_NAME  # [not win]
     - conda inspect objects $PKG_NAME  # [osx]
+    - dot -Tsvg sample.dot
```

Comment:
Maybe we can move `svg` into `run_test.sh` and `run_test.bat`?

Reply:
You mean that last line with the `dot` command? May I ask why?

Reply:
This appears to already be tested. Should we have a better test?

ref: https://github.com/conda-forge/graphviz-feedstock/blob/046c1d35d8c8f6042d253d74220a323c1c5b388b/recipe/run_test.sh#L5
ref: https://github.com/conda-forge/graphviz-feedstock/blob/046c1d35d8c8f6042d253d74220a323c1c5b388b/recipe/run_test.bat#L1